docker/sandbox_generator.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/python3
+import os
+import yaml
+import glob
+import shutil
+import re
+import argparse
+import boto3
+
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+
+
+def download_file(obj, file):
```

Comment:
Would be neater if we had a new file called `s3_utils.py` with the contents of `upload_s3.py` and this function (and whatever is s3 related), removing then `upload_s3.py`